extras/docker/nest-2.12.0/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM nest/docker-master:latest
+
+ARG WITH_MPI
+ARG WITH_GSL
+ARG WITH_MUSIC
+ARG WITH_LIBNEUROSIM
+
+# add user 'nest'
+RUN adduser --disabled-login --gecos 'NEST' --home /home/nest nest && \
+    adduser nest sudo
+
+# Install NEST 2.14.0
+RUN mkdir data && \
+    chown -R nest:nest /home/nest
+
+WORKDIR /home/nest
+
+USER nest
+
+RUN wget https://github.com/nest/nest-simulator/releases/download/v2.12.0/nest-2.12.0.tar.gz && \
+    tar -xvzf nest-2.12.0.tar.gz && \
+    mkdir /home/nest/nest-build && \
+    mkdir /home/nest/nest-install
+
+WORKDIR /home/nest/nest-build
+
+# Install NEST 2.12.0
+
+RUN cmake -DCMAKE_INSTALL_PREFIX:PATH=/home/nest/nest-install \
+    -Dwith-python=2 \
```

Comment:
Isn't there a python3 environment via conda from nest/docker-master:latest? Why not use that?